sov-modules/sov-modules-api/src/dispatch.rs

```diff
@@ -3,12 +3,9 @@ use crate::{CallResponse, Context, Error, QueryResponse, Spec};
 /// Methods from this trait should be called only once during the rollup deployment.
 pub trait Genesis {
     type Context: Context;
-    type Config;
 
     /// Initializes the state of the rollup.
-    fn genesis(
-        config: Self::Config,
-    ) -> Result<<<Self as Genesis>::Context as Spec>::Storage, Error>;
+    fn genesis(storage: <<Self as Genesis>::Context as Spec>::Storage) -> Result<(), Error>;
```

Comment:
Is there any situation where a user would want to initialize a module with something other than a `Storage` instance? For example, maybe you want to pass in a reference to the `account` or `token` module as well?

Reply:
This will work out of the box. Modules can reference each other with `#[module]` tag; they share the same `Storage` (but manage their own `Prefixes`). Here is a unit test for this:
https://github.com/Sovereign-Labs/sovereign/blob/research/sov-modules/sov-modules-macros/tests/module_info/mod_and_state.rs#L31 

I will add more tests for this.

Reply:
Oh, great! Nevermind then!